# Generated by Django 3.1.2 on 2020-10-28 17:42

import django.contrib.auth.models
import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    initial = True

    dependencies = [
    ]

    operations = [
        migrations.CreateModel(
            name='User',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('password', models.CharField(max_length=128, verbose_name='password')),
                ('username', models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='username')),
                ('nickname', models.CharField(max_length=30, unique=True, verbose_name='Nickname')),
                ('playcount', models.IntegerField(default=0, verbose_name='플레이수')),
                ('wincount', models.IntegerField(default=0, verbose_name='이긴수')),
                ('score', models.IntegerField(default=0, verbose_name='점수')),
            ],
            options={
                'db_table': 'User',
                'managed': False,
            },
            managers=[
                ('objects', django.contrib.auth.models.UserManager()),
            ],
        ),
        migrations.CreateModel(
            name='Userconnect',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
            ],
            options={
                'db_table': 'UserConnect',
                'managed': False,
            },
        ),
    ]
